### ISSUE TITLE
Comment contained inside text is removed.

### ISSUE THREAD

**Issue by EricABC** [systemjs/systemjs#1605]
Discovered using systemjs-builder for angular 4, this line here in angular is an example that causes an angular build to fail:
https://github.com/angular/angular/blob/0fa3895d5bdbbc73fe44a3202c6ba8d351fad5da/modules/%40angular/compiler/src/output/output_jit.ts#L18
```js
  const fnBody =
      `${ctx.toSource()}\n//# sourceURL=${sourceUrl}\n${ctx.toSourceMapGenerator().toJsComment()}`;
```
becomes
```js
const /** @type {?} */fnBody = `${ctx.toSource()}\n
```

Version: Although i have systemjs 0.20.9, i believe builder is still using 0.19.46.

**Comment by guybedford** [comment]
Are you using Rollup in the build? That may be responsible for some bad source map stripping code if I remember correctly.

**Comment by EricABC** [comment]
Nope, just appending angular 4 packages together.  https://gist.github.com/EricABC/485afeab06e976b76a224063c181a0fe is the build script for it (including workaround).  The interesting part is at the bottom.

**Comment by guybedford** [comment]
Did you try setting `rollup: false` in the systemjs builder options?

**Comment by EricABC** [comment]
Oh, I see, builder uses rollup.  I'll give that a shot in a few hours.

**Comment by EricABC** [comment]
Ok, I gave it a try to no avail.  From what I see rollup is only used for buildStatic, so it never gets to the part where it looks for the rollup config option, since I'm using bundle.

Here is a simple repro.  Its line 37149 on the output.

```json
{
  "version": "1.0.0",
  "name": "stuff",
  "private": true,
  "dependencies": {
    "rxjs": "^5.2.0",
    "systemjs-builder": "^0.16.3",
    "@angular/core": "4.0.0-rc.2",
    "@angular/compiler": "4.0.0-rc.2"
  }
}
```
```js
#!/usr/bin/env node
var builderFactory = require("systemjs-builder");
var config = {
    baseURL: '/',
    transpiler: false,
    packages: {
        'rxjs': { main: 'Rx.js', defaultExtension: 'js', format: 'cjs' },
        '@angular/core': { defaultExtension: 'js', format: 'esm' },
        '@angular/compiler': { defaultextension: 'js', format: 'esm' }
    },
    paths: {
        "@angular/core": "@angular/core/@angular/core.js",
        "@angular/compiler": "@angular/compiler/@angular/compiler.js"
    }
};
var builder = new builderFactory('/node_modules', config);
return builder.bundle("@angular/compiler", "testing123.js", { sourceMaps: 'inline', rollup: false });
```

**Comment by guybedford** [comment]
Ok, I finally tracked this down and fixed in https://github.com/systemjs/builder/commit/49a34ce879c69f57dbdb2198fbacb118329b4569.

**Comment by guybedford** [comment]
Released in 0.16.4.

**Comment by pcasaes** [comment]
Hi @guybedford

**Comment by pcasaes** [comment]
Hi @guybedford is it possible to add this fix to 0.15? Perhaps release a 0.15.37?
